### DISCUSSION TITLE
Option for parentheses around equation numbers

### DISCUSSION THREAD

**Discussion by jordan-melendez** [quarto-dev/quarto-cli#2413] [Feature Requests]
In certain journals it is common to format references to equations like `Eq. (1.1)` with parentheses around the equation number. Quarto supports modifying `Eq.` with `eq-prefix`, but would it be possible to either

1. Add a parameter similar to `eq-prefix` that allows one to customize the reference further, and/or
2. Ensure that, when compiled to PDF, `\eqref` gets used rather than `\ref` for equation references. This would require `amsmath` I suppose, but would automatically add parentheses and permit further customization.

Thanks!

**Comment by cderv** [comment]
Thanks for your feedback. 

Currently there is no way to control that. I'll open issue about this.

For now, if you target PDF only, you could pass `\eqref` directly bypassing the cross ref labelling for Quarto

Example 
````markdown
---
format: pdf
---

Black-Scholes Eq. \eqref{eq-black-scholes} is a mathematical model that seeks to explain the behavior of financial derivatives, most commonly options:

$$
\frac{\partial \mathrm C}{ \partial \mathrm t } + \frac{1}{2}\sigma^{2} \mathrm S^{2}
\frac{\partial^{2} \mathrm C}{\partial \mathrm C^2}
  + \mathrm r \mathrm S \frac{\partial \mathrm C}{\partial \mathrm S}\ =
  \mathrm r \mathrm C 
$$ {#eq-black-scholes}
```` 

would produce 
![image](https://user-images.githubusercontent.com/6791940/190156098-c502b3ed-cb33-4e52-a6a2-67ecfe9e0e2d.png)

This is not ideal, as not portable in HTML (maybe portability can be added with Lua filter, but unsure). 

It is just a workaround for PDF only, to show it works, but far from ideal. We'll see if we can improve this in the future

**Comment by mcanouil** [comment]
Closing in favour of https://github.com/quarto-dev/quarto-cli/issues/2439
